src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -124,4 +127,37 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
-}
\ No newline at end of file
+    @GET
+    @Path("{instanceName}/team/{wantedTeamName}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response isAdmin(@PathParam("instanceName") String instanceName, @PathParam("teamName") String wantedTeamName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        JsonObject body = new JsonObject();
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        TeamService teamService = new TeamService(client);
+
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            if (wantedTeamName.equals(orgTeam.getName())) {
+                List<User> kabaneroTeamMembers = teamService.getMembers(orgTeam.getId());
```

Comment:
I would just return this List of Users to the API no need to parse anything from it here. jaxrs should be able to automatically convert it to JSON.